# This message describes the O-79 imaging radar tracking filter parameters.

float32 update_rate
float32 min_time
float32 timeout
float32 confidence_level
float32 proc_noise_speed
float32 proc_noise_azim
float32 proc_noise_elev
float32 min_range
float32 max_range
float32 proc_noise_vel_x
float32 proc_noise_vel_y
float32 proc_noise_vel_z
float32 meas_noise_speed
float32 meas_noise_pos
int8 filter_type